### ISSUE TITLE
Discovered hypervisors agent interface is mapped to the controller hosts interface

### ISSUE THREAD

**Issue by jord-bh** [linuxpolska/zabbix-openstack#8]
Perhaps this is a limitation of the script in some way, but it doesn't feel correct that the automatically discovered hypervisors has their agent interfaced mapped towards the controller host which initiated the discovery?

To my understanding, the agent interface is the definition of where zabbix connects to poll for information. In my case, on the discovered hypervisor this is the same as the configured zabbix controller host item.

**Comment by jord-bh** [comment]
Could this be due to Zabbix limitations?

https://support.zabbix.com/browse/ZBXNEXT-2072

**Comment by cloudowski** [comment]
Yes, it looks like it is. Actually I'm still waiting for this issue to be resolved in Zabbix, but it doesn't seem to be an interesting topics for its developers :(

**Comment by jord-bh** [comment]
Too bad, it definitely hinders the usefulness of the setup. iirc from back then, I tried toying around with modifying it afterwards so that the agent interface field would be populated with the correct string, but discovered rules are supposed to stay the way they were created upon discovery. I even went as far as hacking straight in the database but without any luck, so an actual fix from the developers is the only solution as I see it.
